index.html

```diff
@@ -519,19 +524,19 @@ <h2>Signature Suites</h2>
         <dd>
 A URL that identifies the <a>canonicalization algorithm</a> to use on the
 <var>document</var>. For example:
-<code>https://w3id.org/security#GCA2015</code>.
+<code>https://w3id.org/security#URDNA2015</code>.
         </dd>
         <dt>digestAlgorithm</dt>
         <dd>
 A URL that identifies the <a>message digest algorithm</a> to use on the
 <var>canonicalized document</var>. For example:
-<code>https://www.ietf.org/assignments/jwa-parameters#SHA256</code>
+<code>http://w3id.org/digests#sha512</code>
         </dd>
         <dt>signatureAlgorithm</dt>
         <dd>
 A URL that identifies the <a>signature algorithm</a> to use on the
 <var>data to be signed</var>. For example:
-<code>https://www.ietf.org/assignments/jwa-parameters#RS256</code>
+<code>http://w3id.org/security#ed25519</code>
```

Comment:
This doesn't seem right -- we'll want to revisit; I'm not sure what the proper URL is.

Reply:
Np, reverted; we'll revisit the algorithm links shortly.